src/mudlet.pro

```diff
@@ -172,15 +172,14 @@ macx {
 DEFINES += APP_TARGET=\\\"$${TARGET}$${TARGET_EXT}\\\"
 
 ################## DejuVu and Ubuntu Fonts inclusion detection #################
-# Setting an environmental variable WITH_FONTS to the case insensitive value
-# "no" will stop the inclusion of the fonts present in the source code in the
-# compiled binary. Linux packagers or other parties may not want to include the
-# fonts since such resources may already be available in other ways or are not
-# wanted in their binaries.
-# Note: as WITH_FONTS could be a number, a string, something else (or not even
-# exist) we need to be careful in checking it exists before doing much else
-# with it. Also as an environmental variable it is tricky to handle unless we
-# read it into a qmake variable first:
+# To skip bundling Bitstream Vera Sans and Ubuntu Mono fonts with Mudlet,
+# set the environment WITH_FONTS variable to "NO"
+# ie: export WITH_UPDATER="NO" qmake
+# 
+# Note for Mudlet developers: as WITH_FONTS could be a number, a string, 
+# something else (or not even # exist) we need to be careful in checking it
+# exists before doing much else with it. Also as an environmental variable it
+# is tricky to handle unless we read it into a qmake variable first:
 FONT_TEST = $$(WITH_FONTS)
 isEmpty( FONT_TEST ) | !equals($$upper(FONT_TEST), "NO" ) {
```

Comment:
Um, this (and the corresponding one in the other section) is what makes it case in-sensitive - as I said it is - isn't it?

Reply:
My comment on Discord:
> Oh, I got it... this is a bit nasty, tbh: upper wants a value, so it makes UPDATER_TEST (the variable name) upper case, no change here. isEqual wants a variable name as first argument and a value as second argument. So it gets UPDATE_TEST as first argument and compares the value in that variable to NO

Reply:
Sure - I'm mainly interested in making it simple to understand what do you need to do to disable it. I think the current text is trying to say so much, you don't understand anything at all.